src/main/java/com/nology/synergyapi/service/UserContactsService.java

```diff
@@ -41,12 +40,19 @@ public List<UserContactBank> getUserContactsWithBank(Long userId) {
             }
             UserContactBank userContact = new UserContactBank(user.getFirstName(), user.getLastName(),
                     bank.getBankName(), bank.getBankLogo(), 
-                    account.getSortCode(), account.getIBAN(), account.getAccountID(), account.getCurrencyID(), account.getAccountType());
+                    account.getSortCode(), account.getIBAN(), account.getAccountID(), account.getCurrencyID(), account.getAccountType(), user.getuserID());
             userContacts.add(userContact);
        });
 
        return userContacts;
     }
+    public UserContact deleteUserContact(Long userID, Long contactID) {
```

Comment:
Please check if the deleteUserContact that I just added in UserService will satisfy this requirement.